properties.schema

```diff
@@ -1,215 +1,262 @@
 {
-  "type":"object",
+  "type": "object",
   "$schema": "http://json-schema.org/draft-04/schema",
   "id": "http://jsonschema.net",
-  "required":false,
-  "properties":{
+  "required": false,
+  "properties": {
     "pluginLocations": {
-      "type":"object",
-      "required":true,
-      "properties":{
+      "type": "object",
+      "required": true,
+      "properties": {
         "config": {
-          "type":"object"
+          "type": "object"
         },
         "course": {
-          "type":"object",
-          "properties":{
+          "type": "object",
+          "properties": {
             "_assessment": {
-              "type":"object",
-              "required":false,
+              "type": "object",
+              "required": false,
               "legend": "Assessment",
-              "properties":{
+              "properties": {
                 "_postTotalScoreToLms": {
-                  "type":"boolean",
-                  "required":true,
+                  "type": "boolean",
+                  "required": true,
                   "title": "Send Total Score to LMS",
-                  "inputType": { "type": "Boolean", "options": [false, true]},
+                  "inputType": {
+                    "type": "Boolean",
+                    "options": [false, true]
+                  },
+                  "help": "If set to 'true', an overall assessment score will be submitted to the LMS",
                   "validators": []
                 },
                 "_isPercentageBased": {
-                  "type":"boolean",
-                  "required":true,
+                  "type": "boolean",
+                  "required": true,
                   "title": "Percentage based",
-                  "inputType": { "type": "Boolean", "options": [false, true]},
+                  "inputType": {
+                    "type": "Boolean",
+                    "options": [false, true]
+                  },
+                  "help": "If set to 'true', the assessment score will be converted to a percentage.  If 'false', a number will be sent.",
```

Comment:
Picky I know, but a percentage is also a number ;-) Perhaps just leave out the last sentence as it's implied anyway.